modules/tinymce/src/themes/silver/test/ts/browser/editor/NoticationManagerImplTest.ts

```diff
@@ -196,17 +196,17 @@ UnitTest.asynctest('NotificationManagerImpl test', (success, failure) => {
             NamedChain.write('nWarn', cOpenNotification(editor, 'warning', 'Message 2')),
 
             // Check items are positioned so that they are stacked
-            NamedChain.direct('nError', cAssertPosition('Error notification', 220, -383), '_'),
-            NamedChain.direct('nWarn', cAssertPosition('Warning notification', 220, -335), '_'),
+            NamedChain.direct('nError', cAssertPosition('Error notification', 220, -399), '_'),
+            NamedChain.direct('nWarn', cAssertPosition('Warning notification', 220, -351), '_'),
 
             // Shrink the editor to 300px
             NamedChain.direct('body', UiFinder.cFindIn('.tox-statusbar__resize-handle'), 'resizeHandle'),
             NamedChain.direct('resizeHandle', Mouse.cMouseDown, '_'),
             NamedChain.direct('body', cResizeToPos(600, 400, 600, 300), '_'),
 
             // Check items are positioned so that they are stacked
-            NamedChain.direct('nError', cAssertPosition('Error notification', 220, -283), '_'),
-            NamedChain.direct('nWarn', cAssertPosition('Warning notification', 220, -235), '_'),
+            NamedChain.direct('nError', cAssertPosition('Error notification', 220, -299), '_'),
```

Comment:
If you can represent these numbers (e.g. -299) as an expression of other numbers (e.g. initialY - heightOfSuchAndSuch), that can make it easier for the next person to update the test. Is there an expression you can use for these numbers?

Reply:
I hated this as well, as it's very confusing. However it's like that for all these tests, so not something I want to touch atm.